packages/web-components/src/search-widget/client/components/search-form.svelte

```diff
@@ -88,32 +126,203 @@
   .search-form-input:focus {
     outline: none;
   }
+
+  .search-form-control {
+    margin-top: 8px;
+    text-align: left;
+    font-size: 1rem;
+  }
+
+  .search-form-control-radio {
+    cursor: pointer;
+    display: inline-block;
+    line-height: 1.25;
+  }
+
+  .search-form-advanced-search {
+    background: none;
+    border: none;
+    outline: none;
+    float: right;
+    cursor: pointer;
+  }
+
+  .filter-form {
+    margin: 1em auto;
+  }
+
+  @media screen and (min-width: 560px) {
+    .filter-form {
+      width: 80% !important;
+    }
+
+    .filter-form-container {
+      display: flex;
+      margin-left: -8px;
```

Comment:
Can you please use 0.5em - and accross the file so the spacing scales with the font size